file_dir : /home/duckyu/ws/duck_base/src/path_generator/path/indoor/path1